packages/react-shortcuts/src/ShortcutManager/ShortcutManager.tsx

```diff
@@ -61,6 +61,20 @@ export default class ShortcutManager {
     }
   };
 
+  private allModifiersAreHeld(heldKeys: HeldKey, event: KeyboardEvent) {
+    function hasKeyGroups(groups: HeldKey): groups is HeldKey {
+      return groups.every(key => Array.isArray(key));
+    }
+
+    function keyGroupIsHeld(keyGroup: ModifierKey[]) {
+      return keyGroup.every((key: ModifierKey) => event.getModifierState(key));
+    }
+
+    return hasKeyGroups(heldKeys)
+      ? heldKeys.some(keyGroupIsHeld)
```

Comment:
If this is the case does it make more sense to be strict and error out?

Reply:
What do you mean "error out"? Both options are valid.

Reply:
Ignore, I misread this logic 👍